servicecontrol/troubleshooting.md

```diff
@@ -105,3 +105,11 @@ The internal *FailedMessageRetries* collection must be purged in order to restor
 1. Select the "FailedMessageRetries" collection in the left tree
 1. Delete all documents in the collection
 1. Stop maintenance mode
+
+## Microsoft Defender SmartScreen blocks installer 
+
+All code is code signed, but Microsoft SmartScreen (part of Windows Defender) can considers the certificate "untrusted" and shows a warning when trying to install.
+
+The files are code signed correctly and show "NServiceBus Ltd" as the signee, but Microsoft needs to "build trust in" the certificate before not showing the warning. One of the main inputs is to get users to override the warning and install the files. It is required to override that warning by clicking "Run Anyway" until the NServiceBus code signing certificate gets trusted by Microsoft SmartScreen.
```

Comment:
```suggestion
Although the installer is code signed correctly with a certificate owned by "NServiceBus Ltd", SmartScreen will block it from running until Microsoft has built enough "trust" in the certificate. One of the main inputs to building that trust is when users grant permission to run the installer. To grant permission to run the installer, click "Run Anyway". This will no longer be required when Microsoft decides to trust the certificate.
```